packages/pulsar-updater/src/main.js

```diff
@@ -0,0 +1,213 @@
+const { CompositeDisposable } = require("atom");
+let superagent;
+let findInstallMethod;
+let shell;
+
+class PulsarUpdater {
+  activate() {
+    this.disposables = new CompositeDisposable();
+    this.cache = require("./cache.js");
+
+    this.disposables.add(
+      atom.commands.add("atom-workspace", {
+        "pulsar-updater:check-for-update": () => {
+          this.checkForUpdates();
+        },
+      })
+    );
+    this.disposables.add(
+      atom.commands.add("atom-workspace", {
+        "pulsar-updater:clear-cache": () => {
+          this.cache.empty("last-update-check");
+          this.cache.empty(`installMethod.${atom.getVersion()}`);
+        },
+      })
+    );
```

Comment:
```suggestion
    this.disposables.add(
      atom.commands.add("atom-workspace", {
        "pulsar-updater:check-for-update": () => {
          this.checkForUpdates();
        },
        "pulsar-updater:clear-cache": () => {
          this.cache.empty("last-update-check");
          this.cache.empty(`installMethod.${atom.getVersion()}`);
        },
      })
    );
```

Haven't tested it, but this ought to work.